### PR TITLE
Serverless Support

### PR DESCRIPTION
**Pull request by ericzarcone**
Creates the serverless.yml file to allow this to be deployed to lambda.  Currently hard-coded for the dev account for testing purposes.

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#serverless.yml
```
Comment (gabelerner) [inline serverless.yml:1]: # note to reviewers
this repo is public (a fork, not a private one) so keep it generic

File: serverless.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  stage: ${opt:stage, 'dev'}
+
+functions:
+  bucket-antivirus-update-ez-test:
+    handler: update.lambda_handler
+    name: bucket-antivirus-update
+    description: Updates the antivirus definitions and adds them to the bucket.
+    memorySize: 512 # memorySize for this specific function.
+    timeout: 300
+    runtime: python2.7
```
Comment (gabelerner) [inline serverless.yml]: can move this up to provider, don't need to dup it in all functions

File: serverless.yml [hunk 3]
```diff
@@ -10,6 +10,9 @@ provider:
   stage: ${opt:stage, 'dev'}
   runtime: python2.7
   timeout: 300
+  environment:
+      AV_DEFINITION_S3_BUCKET: ${env:AV_DEFINITION_S3_BUCKET}
+      DATADOG_API_KEY: ${env:DATADOG_API_KEY}
```
Comment (lingrino) [inline serverless.yml]: You could make this env var "optional" by adding a default blank value like `DATADOG_API_KEY: ${env:DATADOG_API_KEY, ""}`

File: serverless.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  region: us-east-2
+  stage: ${opt:stage, 'dev'}
+  runtime: python2.7
+  timeout: 300
+  environment:
+      AV_DEFINITION_S3_BUCKET: ${env:AV_DEFINITION_S3_BUCKET}
+      DATADOG_API_KEY: ${env:DATADOG_API_KEY}
+
+functions:
+  bucket-antivirus-update-ez-test:
```
Comment (gabelerner) [inline serverless.yml]: remove the words `ez-test` from here and below

**Review by knehring — APPROVED** [review]

**Review by lingrino — APPROVED** [review]

**Comment by Kuraiou** [comment]
just curious, what's the status of this?

**Review by natw — APPROVED** [review]

**Review by lingrino — APPROVED** [review]

**Review by afpacket — APPROVED** [review]
Hello friends 😃

**Pull request closed without merging**